external-crates/move/crates/move-package/src/lock_file/schema.rs

```diff
@@ -106,7 +106,7 @@ impl Packages {
         let Schema { move_: packages } =
             toml::de::from_str::<Schema<Packages>>(&contents).context("Deserializing packages")?;
 
-        Ok((packages, read_header(&contents)?))
+        Ok((packages, Header::from_str(&contents)?))
```

Comment:
This is the only place we did schema:read_header outside of the lock file schema, and now it uses schema::Header::read, which now becomes the canonical public interface for reading the lock file header.